apps/settings/serializers/auth/ldap_ha.py

```diff
@@ -85,6 +86,15 @@ class LDAPHASettingSerializer(serializers.Serializer):
     AUTH_LDAP_HA = serializers.BooleanField(required=False, label=_('LDAP HA'))
     AUTH_LDAP_HA_SYNC_ORG_IDS = OrgListField()
 
+    def validate(self, attrs):
+        data = {
+            "is_periodic": attrs['AUTH_LDAP_HA_SYNC_IS_PERIODIC'],
+            "crontab": attrs['AUTH_LDAP_HA_SYNC_CRONTAB'],
+            "interval": attrs['AUTH_LDAP_HA_SYNC_INTERVAL']
+        }
+        PeriodTaskSerializerMixin(data=data).is_valid(raise_exception=True)
+        return super().validate(attrs)
+
     def post_save(self):
         keys = ['AUTH_LDAP_HA_SYNC_IS_PERIODIC', 'AUTH_LDAP_HA_SYNC_INTERVAL', 'AUTH_LDAP_HA_SYNC_CRONTAB']
         kwargs = {k: self.validated_data[k] for k in keys if k in self.validated_data}
```

Comment:
There are no known inconsistencies or issues with the provided code. The function works correctly within its constraints, but I suggest reviewing it closely before implementing to ensure there aren't any errors that were missed during review.

The use of `periodictask` is correct based on its documentation which states:

- `is_periodic`: Whether synchronization takes place periodically (e.g., daily/weekly/hourly/etc.)

If there are other parts of this process requiring verification not covered here, please let us know!

Also, be sure to test all inputs thoroughly against expected outcomes in a controlled environment to catch potential edge cases or regressions.